appengine/flexible/Datastore/Datastore.Sample/Program.cs

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+using Google.Cloud.Datastore.V1;
+using System.Net;
+
+public class Program {
+	private WebApplication App {get; set;}
+	private DatastoreDb Datastore {get; set;}
+	private KeyFactory VisitKeyFactory {get; set;}
+
+	private Program(string[] args) {
+		var builder = WebApplication.CreateBuilder(args);
+		App = builder.Build();
+		App.MapGet("/", handleGetAsync);
+		Datastore = DatastoreDb.Create(builder.Configuration["GoogleProjectId"]);
+		VisitKeyFactory = Datastore.CreateKeyFactory("visit");

```

Comment:
If you want to use current .NET best practices, you'd use DI for the Datastore DB and the key factory, something like this (I'm writing directly on GitHub so this may have syntax errors):
```suggestion
		var builder = WebApplication.CreateBuilder(args);
		var datastore = DatastoreDb.Create(builder.Configuration["GoogleProjectId"]);
		builder.AddSingleton(datastore);
		builder.AddSingleton(datastore.CreateKeyFactory("visit"));
		App = builder.Build();
		App.MapGet("/", HandleGetAsync);
```

And then, on HandleGetAsync, you include those as parameters so they ar injected. See my comment there.

Reply:
Done, I think.